test/transaction.test.ts

```diff
@@ -103,7 +105,26 @@ describe('buildTx', () => {
     });
     assert.doesNotThrow(() => Psbt.fromBase64(unsignedTx));
 
-    const blindedBase64 = await sender.blindPset(unsignedTx, [2]);
+    const blindingDataMap = new Map<number, BlindingDataLike>();
+
+    const transaction = Transaction.fromHex(psetToUnsignedHex(unsignedTx));
+    for (let i = 0; i < transaction.ins.length; i++) {
+      const input = transaction.ins[i];
+      const blindingData = args.unspents.find(
+        u =>
+          input.hash.equals(Buffer.from(u.txid, 'hex').reverse()) &&
+          u.vout === input.index
+      )!.unblindData;
+      blindingDataMap.set(i, blindingData);
+    }
+
+    // blind only the change output
+    const blindedBase64 = await sender.blindPset(
+      unsignedTx,
+      [2],
+      undefined,
+      blindingDataMap
+    );
     const signedBase64 = await sender.signPset(blindedBase64);
     const signedPset = decodePset(signedBase64);
 
```

Comment:
Woudl you try to check on the explorer /tx endpoint if `broadcastTx` has actually done his job?

Reply:
@louisinger